types/tests/index.test-d.ts

```diff
@@ -0,0 +1,15 @@
+import {expectType, expectError} from 'tsd';
+import { HomebaseProvider, useTransact, useEntity, useQuery } from '../index';
+
+expectType<object>(HomebaseProvider({}));
+expectError(HomebaseProvider("a"));
+expectError(HomebaseProvider(1));
+
+expectType<Array<Array<object>>>(useTransact());
```

Comment:
Does this work without wrapping the hooks in HomebaseProvider? I would expect them to need a homebase-context.